packages/ui/pod-pcd-ui/README.md

```diff
@@ -0,0 +1,37 @@
+<p align="center">
```

Comment:
any reason not to include this same (ofc edited where appropriate to match the package) html preamble in the rest of the `README`s you added in this PR?

Reply:
I copied this from the template and existing packages, and the UI package I copied was the only one which had this preamble.  I can try to modify it appropriately.